app/pages/projects.cjsx

```diff
@@ -22,14 +22,12 @@ module.exports = React.createClass
     query = {include: 'avatar'}
 
     if !apiClient.params.admin
-      query.launch_approved = true
+      query.homepage = true
```

Comment:
an api param override to select the fast serializer. there is another that i've got called `?cards=true` as well.